f/php/bootstrap

```diff
@@ -0,0 +1,144 @@
+#!/opt/bin/php -c/opt/php.ini
+<?php
+use Bref\Bridge\Psr7\RequestFactory;
+use Bref\Http\LambdaResponse;
+use Symfony\Component\Process\Process;
+
+echo 'Cold start' . PHP_EOL;
+
+ini_set('display_errors', '1');
+error_reporting(E_ALL);
+
+$lambdaRuntimeApi = getenv('AWS_LAMBDA_RUNTIME_API');
+
+require __DIR__ . '/vendor/autoload.php';
+
+$server = new Process("php -S localhost:8000 'index.php'");
```

Comment:
Oh this should probably be `escapeshellarg(getenv('_HANDLER'))` instead of `'index.php'`, and the Handler in the yaml file needs to be index.php